aea/cli_gui/app.py

```diff
@@ -1,22 +1,43 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Implementation of the CLI_GUI.py."""
+
 import argparse
 import glob
 import os
 import subprocess
 
-
 import connexion
 import flask
+import os.path
+
 
 parser = argparse.ArgumentParser(description='Launch the AEA CLI GUI')
 parser.add_argument(
     '-ad',
     '--agent_dir',
-    default='./',
+    default=os.getcwd(),
     help='Location of script and package files and where agents will be created (default: my_agents)'
 )
 
-args = parser.parse_args()
-
+args = parser.parse_args()  # pragma: no cover
```

Comment:
This fails the tox -e py37 test with the error : 

`pytest: error: unrecognized arguments: --doctest-modules aea tests/ --cov-report=html --cov-report=term --cov=aea
`

Reply:
Good point. I think I solved this issue with this commit:

https://github.com/fetchai/agents-aea/pull/176/commits/2966fdee0f42144539992ea259ccc1d3ad8e4eb3